components/card/index.css

```diff
@@ -35,15 +107,57 @@ governing permissions and limitations under the License.
   flex-direction: column;
 
   box-sizing: border-box;
-  min-inline-size: var(--spectrum-card-min-width); /* setting min-width incase a product overrides width */
+  min-inline-size: var(--mod-card-minimum-width, var(--spectrum-card-minimum-width));
 
-  border: var(--spectrum-card-border-size) solid transparent;
-  border-radius: var(--spectrum-card-border-radius);
+  border: var(--mod-card-border-width, var(--spectrum-card-border-width)) solid transparent;
+  border-radius: var(--mod-card-corner-radius, var(--spectrum-card-corner-radius));
+  border-color: var(--highcontrast-card-border-color, var(--mod-card-border-color, var(--spectrum-card-border-color)));
+  background-color: var(--highcontrast-card-background-color, var(--mod-spectrum-card-background-color, var(--spectrum-card-background-color)));
 
   text-decoration: none;
 
-  &:focus {
+  &::after {
+    content: '';
+    block-size: calc(100% + (var(--mod-card-focus-indicator-gap, var(--spectrum-card-focus-indicator-gap)) * 2));
+    inline-size: calc(100% + (var(--mod-card-focus-indicator-gap, var(--spectrum-card-focus-indicator-gap)) * 2));
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: 0;
```

Comment:
Would you mind changing these to use their [logical counterparts](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties#properties_for_floating_and_positioning)?